### PR TITLE
i18n(pt-BR): Add Prefetch Guide translation

### PR DESCRIPTION
**Pull request by vitorxfs**
#### Description
Add Brazilian Portuguese translation for the Prefetch Guide (`guides/prefetch`)

There are some broken links directing to the prefetch section from `reference/configuration-reference`, which translation is not up to date yet.
I have opened an issue for this problem, so this PR is blocked for now.

#### Related issues & labels

- Blocked by https://github.com/withastro/docs/issues/6177

### CODE REVIEW

File: src/content/docs/pt-br/guides/prefetch.mdx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+```html
+<a href="/about" data-astro-prefetch="false">About</a>
+```
+
+A estratégia de pré-carregamento padrão pode ser alterada para todos os links com `prefetch.defaultStrategy`, como foi mostrado na [seção Estratégia padrão de pré-carregamento](#estratégia-padrão-de-pré-carregamento).
+
+## Pré-carregar programaticamente
+
+Nem sempre a navegação será exibida na forma de links `<a />`, então você pode também pré-carregar páginas programaticamente com a API `prefetch()` do módulo `astro:prefetch`:
```
Comment (Fryuni) [inline src/content/docs/pt-br/guides/prefetch.mdx:91]:
```suggestion
Como nem sempre a navegação será exibida na forma de links `<a />`, você pode também pré-carregar páginas programaticamente com a API `prefetch()` do módulo `astro:prefetch`:
```

"As ..." starting the sentence is the same as "Given that ..."

File: src/content/docs/pt-br/guides/prefetch.mdx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+```js
+// Pré-carregamento com `fetch()` possui prioridade mais alta.
+prefetch('/sobre', { with: 'fetch' });
+
+// Pré-carregamento com `<link rel="prefetch">` possui prioridade mais baixa
+// e é manualmente agendada pelo navegador. (padrão)
+prefetch('/sobre', { with: 'link' });
+```
+
+A API `prefetch()` inclui as mesmas detecções de [modo de economia de dados](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) e [de conexão lenta](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType) de forma que o pré-carregamento ocorre apenas quando é necessário.
```
Comment (Fryuni) [inline src/content/docs/pt-br/guides/prefetch.mdx:117]:
```suggestion
A API `prefetch()` inclui as mesmas detecções de [modo de economia de dados](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) e de [conexão lenta](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType) de forma que o pré-carregamento ocorra apenas quando é necessário.
```

**Review by Fryuni — COMMENTED** [review]
Great work! And congrats for your first Astro Docs PR 🎉 🎉 🎉

Welcome to the stars!

File: src/content/docs/pt-br/guides/prefetch.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
+---
+title: Pré-carregamento
+description: Pré-carregue links para uma navegação mais ágil entre páginas.
+i18nReady: true
+---
+
+O tempo de carregamento das páginas gera grande impacto na usabilidade e no nível de satisfação de um site. A **opção de pré-carregamento** do Astro traz os benefícios de uma navegação quase instantânea à sua aplicação multi-página (MPA) enquanto os seus visitantes interagem com o site.
```
[the remark this answers is not in the document]
Comment (Fryuni) [reply src/content/docs/pt-br/guides/prefetch.mdx:7]: "while your users interact" and the translation "enquanto" would fit as a description for the `load` strategy...

But that is not the default

File: src/content/docs/pt-br/guides/prefetch.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
+---
+title: Pré-carregamento
+description: Pré-carregue links para uma navegação mais ágil entre páginas.
+i18nReady: true
+---
+
+O tempo de carregamento das páginas gera grande impacto na usabilidade e no nível de satisfação de um site. A **opção de pré-carregamento** do Astro traz os benefícios de uma navegação quase instantânea à sua aplicação multi-página (MPA) conforme os seus visitantes interagem com o site.
+
+## Habilitar o Pré-carregamento
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/guides/prefetch.mdx:9]: No original mantemos a entitualação de frase, e.x. "Enable prefetch" ao invés de "Enable Prefetch".
```suggestion
## Habilitar o pré-carregamento
```

File: src/content/docs/pt-br/guides/prefetch.mdx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+<a href="/sobre" data-astro-prefetch>
+```
+
+Perceba que o pré-carregamento funciona apenas para links do seu próprio site, e não para links externos.
+
+## Configuração de Pré-carregamento
+
+A configuração de `prefetch` também aceita um objeto opcional para customizar o pré-carregamento.
+
+### Estratégias de Pré-carregamento
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/guides/prefetch.mdx:33]:
```suggestion
## Configuração de pré-carregamento

A configuração de `prefetch` também aceita um objeto opcional para customizar o pré-carregamento.

### Estratégias de pré-carregamento
```

File: src/content/docs/pt-br/guides/prefetch.mdx
Comment (Yan-Thomas) [inline src/content/docs/pt-br/guides/prefetch.mdx:1]: Apesar do título dentro da página estar traduzido, isso não será refletido na barra lateral (veja como ainda está "Prefetch"), poderia adicionar a tradução no arquivo em `src/i18n/pt-br/nav.ts`? O nosso [Guia de Tradução](https://contribute.docs.astro.build/guides/i18n/#translation-process) explica um pouco mais.

**Review by Yan-Thomas — COMMENTED** [review]
Thanks @vitorxfs, amazing work! I've left a few suggestions for you 🙌

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**